packages/multimedia/ffmpeg/package.mk

```diff
@@ -81,6 +81,12 @@ else
   FFMPEG_FPU="--disable-neon"
 fi
 
+if [ "$TARGET_ARCH" = "x86_64" ]; then
+  FFMPEG_X86ASM="--enable-x86asm"
```

Comment:
`--enable-x86asmexe=yasm` or update depends to nasm?

Reply:
Yes, it looks like `nasm` became the default (instead of `yasm`) in `ffmpeg-3.4`. `ffmpeg` will now use `nasm` if it's installed in the build host, otherwise when `nasm` isn't available it will use `yasm` (as it will find `yasm` in the toolchain). I'll push another commit that changes the default back to `yasm` - many thanks.